scripts/ft-benchmark-data-sender.py

```diff
@@ -121,6 +121,7 @@ def commit_to_db(data: dict) -> None:
     commit_hash, commit_time = get_commit()
     response = {
         "time": time_begin.strftime('%Y-%m-%dT%H:%M:%SZ'),
+        "time_end": datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
@MCJOHN974 can you verify if this is the correct way to add `time_end` to the `response`?

Reply:
Yes I think it will work